src/Silk.Core.Logic/Commands/Miscellaneous/UptimeCommand.cs

```diff
@@ -17,7 +17,7 @@ public class UptimeCommand : BaseCommandModule
         public async Task UpTime(CommandContext ctx)
         {
             DateTime now = DateTime.Now;
-            TimeSpan uptime = now.Subtract(Program.Startup);
+            TimeSpan uptime = now.Subtract(Startup.StartupTime);
```

Comment:
I'm all for descriptive variables but in this case, I think just using `DateTime.Now.Subtract(Startup.StartupTime);` would work.